### ISSUE TITLE
NetHunter Terminal Issue

### ISSUE THREAD

**Issue by MatthewHKnight** [offensive-security/kali-nethunter#551]
### Device:

Nexus 6
### OS version (KitKat/Lollipop/Marshmallow/Nougat):

Marshmallow 6.0.1 build MOB30W
### Built from repo (date and build command) or downloaded from website (links):

2016-08-28 10:28:25 am EDT python build.py -d shamu -m -g ARCH -fs full -ns
## Issue:

NetHunter Terminal when starting it and picking Kali for the option the Terminal will just disappear and not load no way to use the Kali Linux commands in the Kali Terminal is their any fix for this?
Also I noticed after flashing NetHunter on the Nexus 6 with the build I made SuperSU said the Binary needed to be updated but would keep saying Installation failed ! and to reboot rebooted it about 3 times before I re-rooted the phone now SuperSU doesn't say the binary need to be updated however the Kali Terminal will not open it just disappears when you pick Kali from the NetHunter Terminal.

**Comment by akrolla** [comment]
Reinstall kernel is what worked fot me, I downloaded and flashed anykernel from xda

**Comment by akrolla** [comment]
Do that before you have to reinstall the nethunter zip

**Comment by akrolla** [comment]
Unless anybody knows easier?

**Comment by binkybear** [comment]
MatthewHKnight: This is usually caused by a missing "bootkali."  You can test this by going into a androidsu terminal and typing bootkali.  If you are missing bootkali, you need to generate it by running the Nethunter app (you also need supersu installed).  This creates the symlink to be able to boot into kali.

From your comment it sounds like you may want to try flashing supersu manually.

**Comment by akrolla** [comment]
Perhaps from an incomplete chroot during install? I couldn't match shasums
and verify install because of aroma installer freezing after awhile, when I
relieved the stress on aroma by flashing stock,root putting multirom busy
box, then recovery, install anykernel and then nethunter.

On Aug 28, 2016 5:46 PM, "binkybear" <notifications@github.com> wrote:

MatthewHKnight: This is usually caused by a missing "bootkali." You can
test this by going into a androidsu terminal and typing bootkali. If you
are missing bootkali, you need to generate it by running the Nethunter app
(you also need supersu installed). This creates the symlink to be able to
boot into kali.

From your comment it sounds like you may want to try flashing supersu
manually.

—
You are receiving this because you commented.

Reply to this email directly, view it on GitHub
<https://github.com/offensive-security/kali-nethunter/issues/551#issuecomment-243004313>,
or mute the thread

**Comment by MatthewHKnight** [comment]
Just curious if I did this wrong what I did was did a clean stock flash of Android 6.0.1 build MOB30W and then went in and flashed the build I made using the 
`python build.py -d shamu -m -g ARCH -fs full -ns`
was I supposed to use a different Kernel and not the stock one?

**Comment by binkybear** [comment]
I would have done:
python build.py -d shamu -m -fs full

Flash clean stock > Flash TWRP > Install zip file in TWRP

**Comment by MatthewHKnight** [comment]
Device still needs to be rooted first before installing nethunter right?

**Comment by akrolla** [comment]
Yes

**Comment by jcadduono** [comment]
only if the zip is using --nosu (-ns) or older than 3.1, otherwise building from ./build.py -d device -m will install supersu automatically during the install.

**Comment by MatthewHKnight** [comment]
Yeah but for some reason it keeps saying SuperSu needs to be updated but fails not sure I've tried installing without SuperSu and manually installing it but it keeps saying SuperSU is outdated then when I go to update it it fails. and tells me to reboot.

**Comment by secnico** [comment]
Hey.

I have had this same issue but accidentally found a work around. Im using a Nexus 6 and recently flashed Pure 7.1, Gapps, generic and then kernel. What i noticed was...

1. Couldn't access terminal from selection the terminal. So i started messing with running scripts from NH, SuperSU, Hacker keyboard, android keyboard and voice input. 
2. One of the above worked and I initially thought it was when i disabled voice input. When i rebooted the original problem returned and was persistent.
3. I then thought that as it had already worked before this must be something to do with a config' ? Anyway, no luck for a day. but explored the app in more detail and wasn't concerend with terminal
4. Then I remembered running a script. I returned to Mana. Ran it. so far so good. I then stopped it Mana and ran Terminal and it worked. I rebooted and trued again. It worked again. So, i know its a work around but the issue cannot be that serious? Over to you guys...
